pkgs/by-name/ta/taskwarrior3/package.nix

```
@@ -90,6 +93,14 @@ stdenv.mkDerivation (finalAttrs: {
     python3
   ];
 
+  doInstallCheck = true;
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}";
```

```suggestion
  versionCheckProgram = "${placeholder "out"}/bin/task";
```
I remember someone somewhere saying that `finalAttrs.meta.*` is a bad idea for some reason. I can't remember why, or if it's even a anti-pattern in the first place, but do let me know if this should be changed.